drivers/gles3/rasterizer_storage_gles3.cpp

```diff
@@ -6943,6 +6943,10 @@ bool RasterizerStorageGLES3::free(RID p_rid) {
 		glDeleteTextures(1, &cls->distance);
 		canvas_light_shadow_owner.free(p_rid);
 		memdelete(cls);
+	} else if (particles_owner.owns(p_rid)) {
+		Particles *prtcls = particles_owner.get(p_rid);
```

Comment:
Please name the variable in an intelligible way. `particles` is much clearer, we don't need to guess if vowels were removed or if it's a lowercase acronym :)

Reply:
Otherwise nice fix, thanks! :)

Reply:
good point, branch updated :)